frontend/assets/translations/es.json

```diff
@@ -0,0 +1,921 @@
+                        "redirectUrl":"Redirect URL"
+                    },
+                    "github":{
+                        "title":"Github",
+                        "hostName":"Host Name",
+                        "enterHostName":"Enter Host Name",
+                        "requiredGithub":"Required if GitHub is self hosted",
+                        "clientId":"Client Id",
+                        "enterClientId":"Enter Client Id",
+                        "clientSecret":"Client Secret",
+                        "enterClientSecret":"Enter Client Secret",
+                        "encrypted":"Encrypted",
+                        "redirectUrl":"Redirect URL"
+                    },
+                    "passwordLogin":"Password Login",
+                    "environmentVar" : {
+                        "noEnvConfig":"You haven't configured any environment variables, press the 'Add new variable' button to create one",
+                        "envWillBeDeleted":"Variable will be deleted, do you want to continue?",
+                        "addNewVariable":"Add new variable",
+                        "variableForm":{
+                            "addNewVariable":"Add new variable",
+                            "updatevariable":"Update variable",
+                            "name":"Name",
+                            "value":"Value",
+                            "enterVariableName":"Enter Variable Name",
+                            "enterValue":"Enter Value",
+                            "type":"Type",
+                            "enableEncryption":"Enable encryption",
+                            "addVariable":"Add variable"
+                        },
+                        "variableTable":{
+                            "name":"name",
+                            "value":"value",
+                            "type":"type",
+                            "secret":"secret"
+                        }
+                    }
+                }
+            }
+        },
+        "profileSettingPage":{
+            "profileSettings":"Profile Settings",
+            "firstName":"First name",
+            "lastName":"Last name",
+            "enterFirstName":"Enter First Name",
+            "enterLastName":"Enter Last Name",
+            "email":"Email",
+            "avatar":"Avatar",
+            "update":"Update",
+            "profile":"Profile",
+            "changePassword":"Change password",
+            "currentPassword":"Current password",
+            "newPassword":"New password",
+            "confirmNewPassword":"Confirm new password",
+            "enterCurrentPassword":"Enter current password",
+            "enterNewPassword":"Enter new password"
+        },
+        "profile":"Profile",
+        "logout":"Logout"
+    },
+    "homePage":{
+        "appCard":{
+            "changeIcon":"Cambiar el ícono",
+            "addToFolder":"Añadir a la carpeta",
+            "move":"Muevete",
+            "to":"A",
```

Comment:
Where is going to be used this? The user will be moving folders/files from one directory to another? If that is the case, I think will be more apropiate to translate it as `Hacia`.